public/app/features/dashboard/utils/tracking.ts

```diff
@@ -11,13 +11,29 @@ export function trackDashboardLoaded(dashboard: DashboardModel, versionBeforeMig
       return r;
     }, {});
 
+  // Track number and type of deprecated panels still in use
+  const deprecatedGraphPanelCount = dashboard.panels.filter((panel) => panel.type === 'graph').length;
+  const deprecatedTablePanelCount = dashboard.panels.filter((panel) => panel.type === 'table-old').length;
+  const deprecatedStatPanelCount = dashboard.panels.filter((panel) => panel.type === 'grafana-singlestat-panel').length;
+  const deprecatedPieChartPanelCount = dashboard.panels.filter(
+    (panel) => panel.type === 'grafana-piechart-panel'
+  ).length;
+  const deprecatedWorldmapPanelCount = dashboard.panels.filter(
+    (panel) => panel.type === 'grafana-worldmap-panel'
+  ).length;
```

Comment:
I wonder if this would be a performance issue because sometimes the dashboards are huge. What if we use a `reduce` function to return an object with all the deprecated panels and then we deconstruct in the event metadata?